examples/step-87/doc/intro.dox

```diff
@@ -0,0 +1,273 @@
+</i>
+
+<h1>Introduction</h1>
+
+This tutorial presents the advanced point-evaluation functionalities of
+deal.II, specifically useful for evaluating finite element solutions at
+arbitrary points. The underlying finite element mesh can be distributed
+among processes, which makes the operations more involved due to communication
+between processes.
+In the examples discussed in this tutorial, we focus on the particular use case of
+point evaluation for distributed meshes, like parallel::distributed::Triangulation.
+Nevertheless, the application to non-distributed meshes is also possible.
+
+<h3>%Point evaluation</h3>
+
+In the context of the finite element method (FEM), it is a common task to
+query the solution $u$ at an arbitrary point $\boldsymbol{x}_q$ in the
+domain of interest $\Omega$
+@f[
+u(\boldsymbol{x}_q) = \sum_{i} N_i(\boldsymbol{x}_q) u_i  \quad\text{with}\quad i\in[0,n_{\text{dofs}}),
+@f]
+by evaluating the shape functions $N_i$ at this point  together with the corresponding  solution coefficients $u_i$.
+After identification of the cell $K$ where the arbitrary point $\boldsymbol{x}_q$ is inside, the
+transformation between $\boldsymbol{x}_q$ and the corresponding coordinates in the reference cell $\hat{\boldsymbol{x}}_q$ is obtained by the mapping $\boldsymbol{x}_q=\boldsymbol{F}_K(\hat{\boldsymbol{x}}_q)$.  In this setting, the evaluation of the solution at an arbitrary point boils down to a cell-local evaluation
+@f[
+	u(\boldsymbol{x}_q) = \sum_{i} \hat{N}^K_i(\hat{\boldsymbol{x}}_q) u_i^K \quad\text{with}\quad i\in[0,n_{\text{dofs_per_cell}}),
+@f]
+with $\hat{N}^K_i$ being the shape functions defined on the reference cell and $u_i^{K}$ the solution coefficients
+restricted to the cell $K$.
+
+Alternatively to point evaluation, evaluating weak-form (integration) operations of the type
+@f[
+u_i
+=
+\left(N_i(\boldsymbol{x}), u(\boldsymbol{x})\right)_\Omega
+=
+\int_{\Omega} N_i(\boldsymbol{x}) u(\boldsymbol{x}) dx
+=
+\sum_q N_i\left(\boldsymbol{x}_q\right) u\left(\boldsymbol{x}_q\right) |J\left(\boldsymbol{x}_q\right)| w\left(\boldsymbol{x}_q\right) \quad\text{with}\quad i\in[0,n_{\text{dofs}})
+@f]
+is possible, with $\boldsymbol{x}_q$ being quadrature points at arbitrary positions.
+After the values at the quadrature points have been multiplied by the
+integration weights, this operation can be interpreted as the transpose of the
+evaluation. Not surprisingly, such an operation can be also implemented as a cell loop.
+
+<h3>Setup and communication</h3>
+
+To determine the cell $K$ and the reference position $\hat{\boldsymbol x}_q$ within
+the cell for a given point $\boldsymbol{x}_q$ on distributed meshes, deal.II performs
+a two-level-search approach. First, all processes that might possess the point are
+determined ("coarse search"). For this purpose, e.g., a distributed
+tree based on bounding boxes around locally owned domains using "ArborX"
+@cite lebrun2020arborx is applied. After the potentially owning
```

Comment:
Is ArborX actually used inside `RemotePointEvaluation`? It would be good to be more specific here about what algorithm needs to be selected by the user by explicitly writing code in the part below, and what part is done internally in the classes.

Reply:
Yes, ArborX is used (if available) inside `reinit()`. More precisely, it is used in `GridTools::internal::guess_owners_of_entities()`. See also https://github.com/dealii/dealii/pull/15538